server/utils.go

```diff
@@ -24,3 +26,40 @@ func getGPRCVersion() (string, error) {
 	}
 	return out, nil
 }
+
+// the first return value is true if path is created successfully, and is false otherwise.
+func parseDNSOptions(servers, searches []string, path string) (bool, error) {
```

Comment:
@mrunalp , the bool return value is used to mark whether a `resolv.conf` is newly created, so that the caller can decide whether to add the bind mount or not.